src/App.tsx

```diff
@@ -1,27 +1,58 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
-
-// import { getAll, get5First, getRed } from './api/goods';
+import { getAll, get5First, getRed, getRedGoods } from './api/goods';
```

Comment:
Fix it 
<img width="717" alt="image" src="https://github.com/user-attachments/assets/8d04a0d1-6587-46d1-959d-e044d73ddd7d" />